code/rounds/PreRound.cs

```diff
@@ -63,7 +129,14 @@ private static async Task StartRespawnTimer(TTTPlayer player)
 
         public override void OnPlayerSpawn(TTTPlayer player)
         {
-            AddPlayer(player);
+            if (player.IsForcedSpectator)
+            {
+                return;
+            }
+
+            Extensions.Log.Debug($"Added Hands to {player.Client.Name}");
+
+            player.Inventory.TryAdd(new Hands(), true);
```

Comment:
@Alf21 How do we want to handle this, we end up calling `Player.Respawn()` on everyone (including forced spectators) which gives the spectators `Hands()`.

1. Inside of `Respawn()` we handle the `IsForcedSpectator` case, so I think we still want to call `Respawn()` on them
2. We probably want to be able to give items to dead players, so no internal check inside of `TryAdd()`

Reply:
We should just add a check in the TryAdd method to never give a weapon to a non alive player.

Reply:
Done, mentioned you in another comment.